backends-common/cassandra/src/main/java/org/apache/james/backends/cassandra/migration/MigrationTask.java

```diff
@@ -157,7 +170,7 @@ public TaskType type() {
 
     @Override
     public Optional<TaskExecutionDetails.AdditionalInformation> details() {
-        return Optional.of(new AdditionalInformation(target, Clock.systemUTC().instant()));
+        return Optional.of(new AdditionalInformation(target, Clock.systemUTC().instant(), achievedMigrations));
```

Comment:
return an immutable copy of the list of achievedMigrations

Reply:
ok